std/math.d

```diff
@@ -3182,6 +3182,12 @@ real cbrt(real x) @trusted nothrow @nogc
  *      )
  */
 real fabs(real x) @safe pure nothrow @nogc;      /* intrinsic */
+//FIXME
```

Comment:
No. I have already pushed scores of `FIXME`s into phobos. This is normal practise if DMD should be changed first.